v20.2/hot-ranges.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Hot Ranges
+summary: Hot Ranges API for CockroachDB.
+toc: true
+back_to_top: true
+---
+
+The Hot Ranges endpoint returns ranges with active requests for a specified node.
+
+This API will remain stable across minor (patch) releases within a major release.
+
+### Resource
+
+`GET /_status/hotranges`
+
+### Authorization
+
+- TK token is required.
+- A user must have the `admin` role.
+
+### Request Parameters
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| node_id | [string](#cockroach.server.serverpb.HotRangesRequest-string) |  | If left empty, hot ranges for all nodes/stores will be returned. |
+
+#### Example Request
```

Comment:
@mjibson Before I review the current docgen update, can you let me know if the include on these individual pages needs to be a single unbroken block? I have added an `Example Request` and `Example Response` next to their corresponding parameters, which is a pretty standard API docs structure and IMO the most user-friendly.

If it's not possible for me to somehow break up the include to insert these sections, do you think that auto-generating and including an example request & response could actually be part of the template?

Reply:
If that is what you want then we will have to generate separate files for request and response. That's fine.

Reply:
Yes, that would be great. And maybe a separate one for the special parameter types as well?